src/api/app/views/layouts/webui/webui.html.haml

```diff
@@ -76,4 +76,10 @@
 
       -# Collapsible menu shared between top and bottom navigation
       - if User.session
-        = render partial: 'layouts/webui/watchlist'
+        - if feature_enabled?(:new_watchlist)
+          = render WatchlistComponent.new(user: User.session!,
+          project_name: params[:project],
+          package_name: params[:package],
+          bs_request_number: params[:number])
```

Comment:
Sounds to me you want to use `@project`, `@package` etc. instead of passing around params and instantiating the objects again in your component.

Reply:
Good idea, we adapted the code for this.